src/main/java/com/puppycrawl/tools/checkstyle/JavadocDetailNodeParser.java

```diff
@@ -96,6 +101,10 @@
      */
     public ParseStatus parseJavadocAsDetailNode(DetailAST javadocCommentAst) {
         blockCommentLineNumber = javadocCommentAst.getLineNo();
+
+        // Column number of COMMENT_CONTENT + 1 for leading asterisk
+        javadocColumnNumber = javadocCommentAst.getFirstChild().getColumnNo() + 1;
```

Comment:
I do not like idea to use class field to keep shift for first line only.

please create special method "adjustFirstLineToJavadocIndent(tree, javadocColumnNumber)" and call it after
`final DetailNode tree = convertParseTreeToDetailNode(parseTree);`
No field is required.
The less fields class has the easier to maintain it. Field is like global variable - should be avoided is possible. It is not performance critical place in code so I see no reason in extra field.